src/RPCServer.ts

```diff
@@ -77,7 +72,7 @@ class RPCServer extends EventTarget {
    * the handler to handle timeout before it is forced to end. Defaults to 2,000 milliseconds.
    * @param obj.logger
    */
-  public static async createRPCServer({
+  public static async startRPCServer({
     manifest,
     middlewareFactory = rpcUtilsMiddleware.defaultServerMiddlewareWrapper(),
     handlerTimeoutTime = Infinity, // 1 minute
```

Comment:
The `StartStop` pattern doesn't use a static async factory like this, this needs to be fully removed.

Start stop is used by doing
```ts
const thing = new thing();
await thing.start();
```

Usage of the `RPCServer` needs to be updated in tests to reflect this.